README.md

```diff
@@ -1,32 +1,109 @@
 # listenbrainz-server
-Server for the ListenBrainz project
 
-## Development
+Server for the ListenBrainz project.
 
-Install ChefDK, then:
+
+## Installation
+
+*These instructions are meant to get you started quickly with development
+process. Installation process in production environment might be different.*
+
+First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
 
     $ kitchen create    # creates vm for first time
     $ kitchen converge  # provisions/installs from chef cookbooks
     $ kitchen login     # ssh in to vm
 
-## Webserver
+### Database installation
+
+Since chef doesn't install PostgreSQL and initialize database yet, this has to
+be done manually. First, after you are logged into the virtual machine, install
+`postgresql-9.4` package:
+
+    $ sudo apt-get install postgresql-9.4
+
+After PostgreSQL is installed, add this line at the beginning of
+*/etc/postgresql/9.4/main/pg_hba.conf* file:
+
+    local all all trust
```

Comment:
I just tried setting only that one to trust and it didn't work. When I set the others to trust too it works.